frontend/coprs_frontend/coprs/views/webhooks_ns/webhooks_general.py

```diff
@@ -91,7 +92,8 @@ def add_webhook_history_record(webhook_uuid, user_agent='Not Set',
         log.debug("No build initiated. Webhook not logged to db.")
         return
 
-    webhookRecord = models.WebhookHistory(webhook_uuid=webhook_uuid,
+    webhookRecord = models.WebhookHistory(created_on=int(time.time()),
```

Comment:
@praiskup would you expect to save the timestamp of when we received the webhook, or would you expect to have a timestamp parsed from the webhook itself?

Reply:
Not sure, do we want to have both?  I think the one "when it was delivered to us" is more important because we can be curious in which order they were delivered.  But no strong preference.